extension/Diagnosis.StructureDefinition.json

```diff
@@ -47,25 +44,35 @@
             {
               "type": "value",
               "path": "version"
+            },
+            {
+              "type": "value",
+              "path": "$this"
```

Comment:
Necessary since last slice (no-icd) is bound to value set and does not have fixed URI